### PR TITLE
[ APP ] Add VGG 16 Test Configuration

### PR DESCRIPTION
**Pull request by jijoongmoon**
This PR includes configuration file (vgg.ini) for VGG 16 with bn layer
which is:

 conv3_64 - conv3_64, max_pooling, conv3_128 - conv3_128, max_pooling,
 conv3_256 - conv3_256 - conv3_256, max_pooling, conv3_512 - conv3_512
 - conv3_512, max_pooling, conv3_512 - conv3_512 - conv3_512,
 bn_layer,  max_pooling, fc_4096, bn_layer, fc_4096, bn_layer, fc_100,
 softmax

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

File: Applications/VGG/res/vgg.ini [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+# Layer Section : Name
+[inputlayer]
+Type = input
+Input_Shape = 3:32:32
+
+# Layer Section : Name
+[conv2d_c1_layer]
+Type = conv2d
+kernel_size = 3,3
+bias_initializer=zeros
```
Comment (kparichay) [inline Applications/VGG/res/vgg.ini:29]: this is already done, but for the future, can let the default initializer handle many things like `bias_initializer`, `padding`, `stride` to reduce the ini file size and for readability.

File: Applications/VGG/res/vgg.ini [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+stride =2,2
+padding = 0,0
+pooling = max
+
+[flatten]
+Type=flatten
+
+[fc_f1_layer]
+Type = fully_connected
+Unit = 256
```
Comment (kparichay) [inline Applications/VGG/res/vgg.ini:201]: it's 256 here
however, comments say it to be 4096.

File: Applications/VGG/res/vgg.ini [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+
+[bn_normalization_b2_layer]
+Type = batch_normalization
+epsilon = 1.0e-6
+momentum = 0.9
+Activation = relu
+
+[fc_f2_layer]
+Type = fully_connected
+Unit = 256
```
Comment (kparichay) [inline Applications/VGG/res/vgg.ini:213]: ditto

File: Applications/VGG/res/vgg.ini [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+pool_size = 2,2
+stride =2,2
+padding = 0,0
+pooling = max
+
+[flatten]
+Type=flatten
+
+[fc_f1_layer]
+Type = fully_connected
```
Comment (kparichay) [inline Applications/VGG/res/vgg.ini:200]: for the cifar100, what will be the dimension hereafter flatten for a reference?

File: Applications/VGG/res/vgg.ini [hunk 5]
```diff
@@ -0,0 +1,228 @@
...
+
+[fc_f2_layer]
+Type = fully_connected
+Unit = 256
+weight_initializer = xavier_uniform
+bias_initializer = zeros
+
+[bn_normalization_b3_layer]
+Type = batch_normalization
+epsilon = 1.0e-6
```
Comment (zhoonit) [inline Applications/VGG/res/vgg.ini:219]: Could you add `*_initializer` in bn_layer for the reference?

File: Applications/VGG/res/vgg.ini (hunk 2 above)
Comment (jijoongmoon) [reply Applications/VGG/res/vgg.ini:201]: Yes. but I thought it is too big considering image size.

File: Applications/VGG/res/vgg.ini (hunk 4 above)
Comment (jijoongmoon) [reply Applications/VGG/res/vgg.ini:200]: the dimension will be 512 when it is flattened.

File: Applications/VGG/res/vgg.ini (hunk 5 above)
Comment (jijoongmoon) [reply Applications/VGG/res/vgg.ini:219]: I will.

File: Applications/VGG/res/vgg.ini [hunk 6]
```diff
@@ -0,0 +1,243 @@
...
+filters = 512
+stride = 1,1
+padding = 1,1
+
+[bn_normalization_b1_layer]
+Type = batch_normalization
+epsilon = 1.0e-6
+momentum = 0.9
+Activation=relu
+beta_initializer = zeros
```
Comment (kparichay) [inline Applications/VGG/res/vgg.ini:191]: These should be set and exposed outside.

**Review by kparichay — APPROVED** [review]
LGTM

**Review by zhoonit — APPROVED** [review]
LGTM

File: Applications/VGG/res/vgg.ini (hunk 6 above)
Comment (zhoonit) [reply Applications/VGG/res/vgg.ini:191]: Yes, #548 is the corresponding patch

**Pull request merged**